src/pages/Dashboard/Activities/index.tsx

```diff
@@ -108,9 +143,27 @@ const ActivityDashboard = (): JSX.Element => {
   };
 
   useEffect(() => {
-    void (async function getAllEmployees() {
+    void (async function getAllActivities() {
       await getActivities(businessId ?? '')
+        // @ts-expect-error
         .then(setActivities)
+        .then(async (res) => {
+          // @ts-expect-error
+          const { activities } = res;
+          const activityArray = activities.map((activity) => ({
+            id: activity.id,
+            // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+            title: activity.title,
+            startTime: (activity.startTime).substring(0, 10),
+            endTime: (activity.endTime).substring(0, 10),
+            address: activity.address,
+            description: activity.description,
+            costPerIndividual: activity.costPerIndividual,
+            costPerGroup: activity.costPerGroup,
+            groupSize: activity.groupSize,
+          }));
+          setActivities(activityArray);
+        })
```

Comment:
This has already been done in the getActivities method of the apiLayer file called at line 147. The activities are then loaded into state on line 149. [view in file](https://github.com/Chrysalis-Papilio/Soen490-Papilio-Website/blob/959eda13e97a56622937453f0e29e66f47edaf9c/src/api/apiLayer.tsx#L43-L59)